src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.cpp

```diff
@@ -201,12 +201,13 @@ VTKPolyDataWriterInterface::getSurface( FaceElementSubRegion const & esr,
 }
 std::pair< vtkSmartPointer< vtkPoints >, vtkSmartPointer< vtkCellArray > >
 VTKPolyDataWriterInterface::getEmbeddedSurface( EmbeddedSurfaceSubRegion const & esr,
-                                                NodeManager const & nodeManager ) const
+                                                EmbeddedSurfaceNodeManager const & nodeManager ) const
```

Comment:
So did this add a new `EmbeddedSurfaceNodeManager`? I thought you were trying to unify this with the `NodeManager`?